grails-app/init/sds/webapp2/BootStrap.groovy

```diff
@@ -0,0 +1,43 @@
+package sds.webapp2
+
+import au.org.ala.sds.model.ConservationInstance
+import au.org.ala.sds.model.PlantPestInstance
+import au.org.ala.sds.model.SensitivityCategory
+import au.org.ala.sds.model.SensitivityInstance
+import au.org.ala.sds.model.SensitivityZone
+import grails.converters.JSON
+
+class BootStrap {
+
+    def init = { servletContext ->
```

Comment:
Would be nice to have some code comments here explaining why this is needed and what it does.

Reply:
Commented